Linux-Application/DomesdayDuplicator/usbcapture.cpp

```diff
@@ -472,11 +469,10 @@ void UsbCapture::allocateDiskBuffers(void)
             }
 
             // Lock the buffer into memory, preventing it from being paged out
-#ifdef __linux__
-            if (tryMlock && mlock(diskBuffers[bufferNumber], TRANSFERSIZE * TRANSFERSPERDISKBUFFER) == -1) {
-#elif _WIN32
+#ifdef _WIN32
             if (tryMlock && VirtualLock(diskBuffers[bufferNumber], TRANSFERSIZE * TRANSFERSPERDISKBUFFER) == -1) {
```

Comment:
Looks good except for the return value check here - the docs say `VirtualLock` returns 0 on failure, so this should be `== 0`?

Reply:
That's an embarrassing mistake.  Thanks for catching this.